New failing test_function_returns_input_inner_requires_grad related tests.

### 🐛 Describe the bug

New failing test cases appeared in the latest Nightly / Build-from-source and in some pull workflows:
https://github.com/intel/torch-xpu-ops/actions/runs/25439279276

What is interesting, that those are the same cases for both XPU and CPU.

Cases:
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_True_save_for_jvp_save_tensors_output_mark_dirty_False_cpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_True_save_for_vjp_save_tensors_input_mark_dirty_False_cpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_True_save_for_vjp_save_tensors_output_mark_dirty_False_cpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_False_save_for_jvp_save_tensors_input_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_False_save_for_jvp_save_tensors_output_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_False_save_for_vjp_save_tensors_input_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_False_save_for_vjp_save_tensors_output_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_True_save_for_jvp_save_tensors_input_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_True_save_for_jvp_save_tensors_output_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_True_save_for_vjp_save_tensors_input_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionXPU,test_function_returns_input_inner_requires_grad_True_save_for_vjp_save_tensors_output_mark_dirty_False_xpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_False_save_for_jvp_save_tensors_input_mark_dirty_False_cpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_False_save_for_jvp_save_tensors_output_mark_dirty_False_cpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_False_save_for_vjp_save_tensors_input_mark_dirty_False_cpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_False_save_for_vjp_save_tensors_output_mark_dirty_False_cpu
op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_eager_transforms_xpu.TestAutogradFunctionCPU,test_function_returns_input_inner_requires_grad_True_save_for_jvp_save_tensors_input_mark_dirty_False_cpu

<details>
<summary>Error log</summary>

```text
2026-05-07T03:18:47.7292455Z _ TestAutogradFunctionXPU.test_function_returns_input_inner_requires_grad_True_save_for_jvp_save_tensors_input_mark_dirty_False_xpu _
2026-05-07T03:18:47.7293082Z [gw8] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
2026-05-07T03:18:47.7294089Z RuntimeError: An input that has been returned as-is as output is being saved for backward. This is not supported if you override setup_context. You should return and save a view of the input instead, e.g. with x.view_as(x) or setup ctx inside the forward function itself.
2026-05-07T03:18:47.7294975Z 
2026-05-07T03:18:47.7295160Z During handling of the above exception, another exception occurred:
2026-05-07T03:18:47.7295414Z 
2026-05-07T03:18:47.7295525Z Traceback (most recent call last):
2026-05-07T03:18:47.7296187Z   File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/functorch/test_eager_transforms_xpu.py", line 1226, in test_function_returns_input
2026-05-07T03:18:47.7296874Z     with self.assertRaisesRegex(RuntimeError, err_msg):
2026-05-07T03:18:47.7297326Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 239, in __exit__
2026-05-07T03:18:47.7297785Z     self._raiseFailure('"{}" does not match "{}"'.format(
2026-05-07T03:18:47.7298263Z   File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 163, in _raiseFailure
2026-05-07T03:18:47.7298737Z     raise self.test_case.failureException(msg)
2026-05-07T03:18:47.7299825Z AssertionError: "A input that has been returned as-is" does not match "An input that has been returned as-is as output is being saved for backward. This is not supported if you override setup_context. You should return and save a view of the input instead, e.g. with x.view_as(x) or setup ctx inside the forward function itself."
2026-05-07T03:18:47.7300799Z 
2026-05-07T03:18:47.7300960Z To execute this test, run the following from the base repo dir:
2026-05-07T03:18:47.7301814Z     PYTORCH_TEST_WITH_SLOW=1 python test/xpu/functorch/test_eager_transforms_xpu.py TestAutogradFunctionXPU.test_function_returns_input_inner_requires_grad_True_save_for_jvp_save_tensors_input_mark_dirty_False_xpu

```

</details>

### Versions

<details>
<summary>Versions</summary>

```text
2026-05-06T23:29:13.1629442Z Collecting environment information...
2026-05-06T23:29:13.1629771Z PyTorch version: 2.13.0a0+git4037885
2026-05-06T23:29:13.1630366Z Is debug build: False
2026-05-06T23:29:13.1630618Z CUDA used to build PyTorch: None
2026-05-06T23:29:13.1630895Z ROCM used to build PyTorch: N/A
2026-05-06T23:29:13.1631072Z 
2026-05-06T23:29:13.1631167Z OS: Ubuntu 24.04.4 LTS (x86_64)
2026-05-06T23:29:13.1631448Z GCC version: (Ubuntu 13.3.0-6ubuntu2~24.04.1) 13.3.0
2026-05-06T23:29:13.1631750Z Clang version: Could not collect
2026-05-06T23:29:13.1632005Z CMake version: version 3.31.6
2026-05-06T23:29:13.1632317Z Libc version: glibc-2.39
2026-05-06T23:29:13.1632460Z 
2026-05-06T23:29:13.1632683Z Python version: 3.10.20 (main, Mar  3 2026, 15:36:40) [GCC 13.3.0] (64-bit runtime)
2026-05-06T23:29:13.1633132Z Python platform: Linux-6.8.0-110-generic-x86_64-with-glibc2.39
2026-05-06T23:29:13.1633470Z Is CUDA available: False
2026-05-06T23:29:13.1633711Z CUDA runtime version: No CUDA
2026-05-06T23:29:13.1634113Z CUDA_MODULE_LOADING set to: N/A
2026-05-06T23:29:13.1634378Z GPU models and configuration: No CUDA
2026-05-06T23:29:13.1634687Z Nvidia driver version: No CUDA
2026-05-06T23:29:13.1634935Z cuDNN version: No CUDA
2026-05-06T23:29:13.1635169Z Is XPU available: True
2026-05-06T23:29:13.1635408Z XPU used to build PyTorch: 20250302
2026-05-06T23:29:13.1635668Z Intel GPU driver version:
2026-05-06T23:29:13.1635914Z * libze1:	1.24.0.0-1146~24.04
2026-05-06T23:29:13.1636177Z * intel-opencl-icd:	25.18.33578.51-1146~24.04
2026-05-06T23:29:13.1636455Z Intel GPU models onboard:
2026-05-06T23:29:13.1636694Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1636943Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1637197Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1637562Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1637817Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1638067Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1638316Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1638559Z * Intel(R) Data Center GPU Max 1550
2026-05-06T23:29:13.1638819Z Intel GPU models detected:
2026-05-06T23:29:13.1640410Z * [0] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-1a00-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1643297Z * [1] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-1a00-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1646225Z * [2] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-2c00-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1651208Z * [3] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-2c00-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1654160Z * [4] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-3c00-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1657015Z * [5] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-3c00-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1659955Z * [6] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-5f00-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1662931Z * [7] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-5f00-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1665817Z * [8] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-8600-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1668613Z * [9] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-8600-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1671429Z * [10] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-9000-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1674243Z * [11] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-9000-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1677093Z * [12] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-9900-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1679891Z * [13] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-9900-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1682758Z * [14] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-e300-000000000001, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1685575Z * [15] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBD5, uuid=8680d50b-2f00-0000-e300-000000000002, driver_version='1.6.33578+51', total_memory=65520MB, local_mem_size=128KB, max_compute_units=512, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=512, gpu_subslice_count=64, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
2026-05-06T23:29:13.1687112Z HIP runtime version: N/A
2026-05-06T23:29:13.1687357Z MIOpen runtime version: N/A
2026-05-06T23:29:13.1687606Z Is XNNPACK available: True
2026-05-06T23:29:13.1687845Z Caching allocator config: N/A
2026-05-06T23:29:13.1688007Z 
2026-05-06T23:29:13.1688083Z CPU:
2026-05-06T23:29:13.1688302Z Architecture:                            x86_64
2026-05-06T23:29:13.1688623Z CPU op-mode(s):                          32-bit, 64-bit
2026-05-06T23:29:13.1688991Z Address sizes:                           52 bits physical, 57 bits virtual
2026-05-06T23:29:13.1689355Z Byte Order:                              Little Endian
2026-05-06T23:29:13.1689659Z CPU(s):                                  192
2026-05-06T23:29:13.1689944Z On-line CPU(s) list:                     0-191
2026-05-06T23:29:13.1690247Z Vendor ID:                               GenuineIntel
2026-05-06T23:29:13.1690586Z Model name:                              Intel(R) Xeon(R) Platinum 8468V
2026-05-06T23:29:13.1690917Z CPU family:                              6
2026-05-06T23:29:13.1691178Z Model:                                   143
2026-05-06T23:29:13.1691450Z Thread(s) per core:                      2
2026-05-06T23:29:13.1691725Z Core(s) per socket:                      48
2026-05-06T23:29:13.1691995Z Socket(s):                               2
2026-05-06T23:29:13.1692255Z Stepping:                                8
2026-05-06T23:29:13.1692524Z CPU(s) scaling MHz:                      99%
2026-05-06T23:29:13.1692802Z CPU max MHz:                             3800.0000
2026-05-06T23:29:13.1693167Z CPU min MHz:                             800.0000
2026-05-06T23:29:13.1693465Z BogoMIPS:                                4800.00
2026-05-06T23:29:13.1697200Z Flags:                                   fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf tsc_known_freq pni pclmulqdq dtes64 monitor ds_cpl smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 cat_l2 cdp_l3 intel_ppin cdp_l2 ssbd mba ibrs ibpb stibp ibrs_enhanced fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb intel_pt avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local split_lock_detect user_shstk avx_vnni avx512_bf16 wbnoinvd dtherm ida arat pln pts hfi avx512vbmi umip pku ospke waitpkg avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg tme avx512_vpopcntdq la57 rdpid bus_lock_detect cldemote movdiri movdir64b enqcmd fsrm md_clear serialize tsxldtrk pconfig arch_lbr ibt amx_bf16 avx512_fp16 amx_tile amx_int8 flush_l1d arch_capabilities ibpb_exit_to_user
2026-05-06T23:29:13.1701182Z L1d cache:                               4.5 MiB (96 instances)
2026-05-06T23:29:13.1701534Z L1i cache:                               3 MiB (96 instances)
2026-05-06T23:29:13.1701872Z L2 cache:                                192 MiB (96 instances)
2026-05-06T23:29:13.1702266Z L3 cache:                                195 MiB (2 instances)
2026-05-06T23:29:13.1702565Z NUMA node(s):                            2
2026-05-06T23:29:13.1702855Z NUMA node0 CPU(s):                       0-47,96-143
2026-05-06T23:29:13.1703165Z NUMA node1 CPU(s):                       48-95,144-191
2026-05-06T23:29:13.1703506Z Vulnerability Gather data sampling:      Not affected
2026-05-06T23:29:13.1703847Z Vulnerability Indirect target selection: Not affected
2026-05-06T23:29:13.1704204Z Vulnerability Itlb multihit:             Not affected
2026-05-06T23:29:13.1704529Z Vulnerability L1tf:                      Not affected
2026-05-06T23:29:13.1704845Z Vulnerability Mds:                       Not affected
2026-05-06T23:29:13.1705155Z Vulnerability Meltdown:                  Not affected
2026-05-06T23:29:13.1705480Z Vulnerability Mmio stale data:           Not affected
2026-05-06T23:29:13.1705813Z Vulnerability Reg file data sampling:    Not affected
2026-05-06T23:29:13.1706143Z Vulnerability Retbleed:                  Not affected
2026-05-06T23:29:13.1706470Z Vulnerability Spec rstack overflow:      Not affected
2026-05-06T23:29:13.1706915Z Vulnerability Spec store bypass:         Mitigation; Speculative Store Bypass disabled via prctl
2026-05-06T23:29:13.1707517Z Vulnerability Spectre v1:                Mitigation; usercopy/swapgs barriers and __user pointer sanitization
2026-05-06T23:29:13.1708231Z Vulnerability Spectre v2:                Mitigation; Enhanced / Automatic IBRS; IBPB conditional; PBRSB-eIBRS SW sequence; BHI BHI_DIS_S
2026-05-06T23:29:13.1708784Z Vulnerability Srbds:                     Not affected
2026-05-06T23:29:13.1709105Z Vulnerability Tsa:                       Not affected
2026-05-06T23:29:13.1709428Z Vulnerability Tsx async abort:           Not affected
2026-05-06T23:29:13.1709819Z Vulnerability Vmscape:                   Mitigation; IBPB before exit to userspace
2026-05-06T23:29:13.1710092Z 
2026-05-06T23:29:13.1710200Z Versions of relevant libraries:
2026-05-06T23:29:13.1710462Z [pip3] dpcpp-cpp-rt==2025.3.2
2026-05-06T23:29:13.1710715Z [pip3] impi-rt==2021.17.2
2026-05-06T23:29:13.1710964Z [pip3] intel-cmplr-lib-rt==2025.3.2
2026-05-06T23:29:13.1711239Z [pip3] intel-cmplr-lib-ur==2025.3.2
2026-05-06T23:29:13.1711501Z [pip3] intel-cmplr-lic-rt==2025.3.2
2026-05-06T23:29:13.1711764Z [pip3] intel-opencl-rt==2025.3.2
2026-05-06T23:29:13.1712087Z [pip3] intel-openmp==2025.3.2
2026-05-06T23:29:13.1712335Z [pip3] intel-pti==0.16.0
2026-05-06T23:29:13.1712581Z [pip3] intel-sycl-rt==2025.3.2
2026-05-06T23:29:13.1712823Z [pip3] mkl==2025.3.1
2026-05-06T23:29:13.1713040Z [pip3] mypy==1.16.0
2026-05-06T23:29:13.1713257Z [pip3] mypy_extensions==1.1.0
2026-05-06T23:29:13.1713496Z [pip3] numpy==1.23.2
2026-05-06T23:29:13.1713716Z [pip3] oneccl==2021.17.2
2026-05-06T23:29:13.1713973Z [pip3] oneccl-devel==2021.17.2
2026-05-06T23:29:13.1714225Z [pip3] onemkl-license==2025.3.1
2026-05-06T23:29:13.1714476Z [pip3] onemkl-sycl-blas==2025.3.1
2026-05-06T23:29:13.1714745Z [pip3] onemkl-sycl-dft==2025.3.1
2026-05-06T23:29:13.1715005Z [pip3] onemkl-sycl-lapack==2025.3.1
2026-05-06T23:29:13.1715271Z [pip3] onemkl-sycl-rng==2025.3.1
2026-05-06T23:29:13.1715526Z [pip3] onemkl-sycl-sparse==2025.3.1
2026-05-06T23:29:13.1715772Z [pip3] onnx==1.21.0
2026-05-06T23:29:13.1715978Z [pip3] onnx-ir==0.1.16
2026-05-06T23:29:13.1716203Z [pip3] onnxscript==0.6.2
2026-05-06T23:29:13.1716436Z [pip3] optree==0.13.0
2026-05-06T23:29:13.1716646Z [pip3] tbb==2022.3.1
2026-05-06T23:29:13.1716853Z [pip3] tcmlib==1.4.1
2026-05-06T23:29:13.1717064Z [pip3] torch==2.13.0a0+git4037885
2026-05-06T23:29:13.1717321Z [pip3] torchao==0.18.0.dev20260506+xpu
2026-05-06T23:29:13.1717581Z [pip3] torchaudio==2.11.0a0+c0cbdb9
2026-05-06T23:29:13.1717835Z [pip3] torchvision==0.27.0a0+499ca51
2026-05-06T23:29:13.1718140Z [pip3] triton-xpu==3.7.1+git21033c4e
2026-05-06T23:29:13.1718393Z [pip3] umf==1.0.3
2026-05-06T23:29:13.1718617Z [conda] No relevant packages
2026-05-06T23:29:13.7029733Z + pip list
2026-05-06T23:29:13.7030063Z + grep -E 'torch|intel'
2026-05-06T23:29:14.0028154Z intel-cmplr-lib-rt        2025.3.2
2026-05-06T23:29:14.0028433Z intel-cmplr-lib-ur        2025.3.2
2026-05-06T23:29:14.0028709Z intel-cmplr-lic-rt        2025.3.2
2026-05-06T23:29:14.0028973Z intel-opencl-rt           2025.3.2
2026-05-06T23:29:14.0029227Z intel-openmp              2025.3.2
2026-05-06T23:29:14.0029474Z intel-pti                 0.16.0
2026-05-06T23:29:14.0029726Z intel-sycl-rt             2025.3.2
2026-05-06T23:29:14.0029991Z torch                     2.13.0a0+git4037885
2026-05-06T23:29:14.0030286Z torchao                   0.18.0.dev20260506+xpu
2026-05-06T23:29:14.0030574Z torchaudio                2.11.0a0+c0cbdb9
2026-05-06T23:29:14.0030850Z torchvision               0.27.0a0+499ca51
```

</details>

--- a/test/xpu/dynamo/test_ctx_manager_xpu.py
+++ b/test/xpu/dynamo/test_ctx_manager_xpu.py
@@ -572,7 +572,7 @@ class CtxManagerTests(torch._dynamo.test_case.TestCase):
         res = opt_fn(x)
         self.assertEqual(ref, res)
         self.assertEqual(cnts.frame_count, 1)
-        self.assertExpectedInline(str(cnts.op_count), """16""")
+        self.assertExpectedInline(str(cnts.op_count), """17""")
 
     @unittest.skipIf(not requires_gpu, "requires cuda or xpu")
     def test_cuda_device(self):

--- a/test/xpu/functorch/test_aot_joint_with_descriptors_xpu.py
+++ b/test/xpu/functorch/test_aot_joint_with_descriptors_xpu.py
@@ -1248,11 +1248,11 @@ class inner_f(torch.nn.Module):
             """\
 ('get_attr', 'repeated_subgraph0', {'mod_name': 'my_mod'})
 [('placeholder', 'arg0_1', {'mod_name': 'my_mod'}), ('call_function', 'sin', {'mod_name': 'bar'}), ('call_function', 'mul', {'mod_name': 'bar'}), ('output', 'output', {'mod_name': 'my_mod'})]
-('call_function', 'invoke_subgraph', {'mod_name': 'my_mod'})
+('call_function', 'invoke_subgraph', {'mod_name': 'my_mod', 'call_id': 1})
 ('call_function', 'getitem', {'mod_name': 'my_mod'})
 ('get_attr', 'repeated_subgraph1', {'mod_name': 'my_mod'})
 [('placeholder', 'arg0_1', {'mod_name': 'my_mod'}), ('placeholder', 'arg1_1', {'mod_name': 'my_mod'}), ('call_function', 'sin', {'mod_name': 'bar'}), ('call_function', 'mul', {'mod_name': 'bar'}), ('call_function', 'mul_1', {'mod_name': 'bar'}), ('call_function', 'cos', {'mod_name': 'bar'}), ('call_function', 'mul_2', {'mod_name': 'bar'}), ('output', 'output', {'mod_name': 'my_mod'})]
-('call_function', 'invoke_subgraph_1', {'mod_name': 'my_mod'})
+('call_function', 'invoke_subgraph_1', {'call_id': 1, 'mod_name': 'my_mod'})
 ('call_function', 'getitem_1', {'mod_name': 'my_mod'})""",
         )
 

--- a/test/xpu/functorch/test_eager_transforms_xpu.py
+++ b/test/xpu/functorch/test_eager_transforms_xpu.py
@@ -1218,7 +1218,7 @@ class TestAutogradFunction(TestCase):
         def fn(x):
             return A.apply(x.clone())
 
-        err_msg = "A input that has been returned as-is"
+        err_msg = "An input that has been returned as-is"
 
         a = torch.tensor(2.0, device=device, requires_grad=inner_requires_grad)
         a_t = torch.tensor(2.0, device=device, requires_grad=inner_requires_grad)

--- a/test/xpu/higher_order_ops/test_invoke_subgraph_xpu.py
+++ b/test/xpu/higher_order_ops/test_invoke_subgraph_xpu.py
@@ -1822,7 +1822,9 @@ class GraphModule(torch.nn.Module):
 class GraphModule(torch.nn.Module):
     def forward(self, primals_1: "f32[8, 8]"):
         partitioned_fw_subgraph_0_0 = self.partitioned_fw_subgraph_0_0
+
         invoke_subgraph_2 = torch.ops.higher_order.invoke_subgraph(partitioned_fw_subgraph_0_0, 'partitioned_fw_subgraph_0_0', primals_1);  partitioned_fw_subgraph_0_0 = primals_1 = None
+
         getitem: "f32[8, 8]" = invoke_subgraph_2[0]
         getitem_1: "f32[8, 8]" = invoke_subgraph_2[1];  invoke_subgraph_2 = None
 
@@ -1843,7 +1845,9 @@ class GraphModule(torch.nn.Module):
 class GraphModule(torch.nn.Module):
     def forward(self, tangents_1: "f32[8, 8]"):
         partitioned_bw_subgraph_0_0 = self.partitioned_bw_subgraph_0_0
+
         invoke_subgraph_3 = torch.ops.higher_order.invoke_subgraph(partitioned_bw_subgraph_0_0, 'partitioned_bw_subgraph_0_0', tangents_1, tangents_1);  partitioned_bw_subgraph_0_0 = tangents_1 = None
+
         getitem_2: "f32[8, 8]" = invoke_subgraph_3[0];  invoke_subgraph_3 = None
         return (getitem_2,)
 
@@ -2015,10 +2019,12 @@ class GraphModule(torch.nn.Module):
 class GraphModule(torch.nn.Module):
     def forward(self, primals_1: "f32[8, 8]", primals_2: "f32[8, 8]"):
         partitioned_fw_subgraph_0_0 = self.partitioned_fw_subgraph_0_0
+
         invoke_subgraph_2 = torch.ops.higher_order.invoke_subgraph(partitioned_fw_subgraph_0_0, 'partitioned_fw_subgraph_0_0', primals_1, primals_2);  partitioned_fw_subgraph_0_0 = primals_1 = primals_2 = None
         getitem_6: "f32[8, 8]" = invoke_subgraph_2[3]
         getitem_5: "f32[8, 8]" = invoke_subgraph_2[2]
         getitem_4: "f32[8, 8]" = invoke_subgraph_2[1]
+
         getitem: "f32[8, 8]" = invoke_subgraph_2[0];  invoke_subgraph_2 = None
         sin: "f32[8, 8]" = torch.ops.aten.sin.default(getitem)
         cos: "f32[8, 8]" = torch.ops.aten.cos.default(getitem);  getitem = None
@@ -2043,7 +2049,9 @@ class GraphModule(torch.nn.Module):
     def forward(self, getitem_6: "f32[8, 8]", getitem_5: "f32[8, 8]", getitem_4: "f32[8, 8]", cos: "f32[8, 8]", tangents_1: "f32[8, 8]"):
         mul: "f32[8, 8]" = torch.ops.aten.mul.Tensor(tangents_1, cos);  tangents_1 = cos = None
         partitioned_bw_subgraph_0_0 = self.partitioned_bw_subgraph_0_0
+
         invoke_subgraph_3 = torch.ops.higher_order.invoke_subgraph(partitioned_bw_subgraph_0_0, 'partitioned_bw_subgraph_0_0', getitem_4, getitem_5, getitem_6, mul);  partitioned_bw_subgraph_0_0 = getitem_4 = getitem_5 = getitem_6 = mul = None
+
         getitem_1: "f32[8, 8]" = invoke_subgraph_3[0]
         getitem_2: "f32[8, 8]" = invoke_subgraph_3[1];  invoke_subgraph_3 = None
         return (getitem_1, getitem_2)

